### DISCUSSION TITLE
How to use breadcrumbs

### DISCUSSION THREAD

**Discussion by kdasVF** [Kitware/trame#400] [Q&A] [answered]
Trying to use breadcrumbs, none of the two below options worked. What am I doing wrong? 

Try 1:
vuetify.VBreadcrumbs(
                        "{{items_list}}",
                        v_if="items_list.length",
                        v_show="items_list.length",
                    )

Try 2:
with vuetify.VBreadcrumbs(v_show="items_list.length"):
    vuetify.VBreadcrumbsItem("{{items_list}}", v_if="items_list.length", )

In both cases, the list is printed without formatting in a ul tag. 

<ul class="v-breadcrumbs theme--light">
[
  "item1",
  "item2",
  "item3"
]
</ul>

**Comment by jourdain** [comment, marked as answer]
See https://v2.vuetifyjs.com/en/api/v-breadcrumbs/#props
Try 
```python
vuetify.VBreadcrumbs(
  items=("items_list", [
     dict(text="item1"),
     dict(text="item2"),
     dict(text="item3"),
  ]),
)
```

**Reply by kdasVF** [reply]
Thank you.
